lona/default_settings.py

```diff
@@ -56,6 +56,7 @@
 # views
 CORE_FRONTEND_VIEW = 'lona.default_views.FrontendView'
 FRONTEND_VIEW = ''
+INITIAL_STATE = {}
```

Comment:
This needs a type annotation to make the linter happy again

Reply:
Fixed